### PR TITLE
Update DNSSEC article

### LINKED ISSUE #195: Article about DS, DNSSEC and SERVFAIL
I would like to write an article about what happens when someone has a DS record at their domain registrar without a corresponding DNSSEC signed zone (which we currently do not provide at DNSimple). This manifests in a SERVFAIL in Google's public resolvers and is something we see from time to time as a support issue.

### PR DESCRIPTION
**Pull request by aeden**
Adds a section about automating key rotation with other registrars. Clarifies that rotation means removing the old DS record.

Closes #195

### CODE REVIEW

File: content/articles/dnssec.markdown [hunk 1]
```diff
@@ -72,21 +72,29 @@ Click on the Delete DNSSEC Configuration button to remove the zone signing and t
...
 
 During the key rotation, both the old and new keys are attached to your zone for 3 days (the duration of the key rotation period). At the end of the rotation period our system removes the old key, leaving only the new key in place.
 
+### Automating key rotation
+
+If your domain registrar provides an API for managing DS records, you can automate rotation for domains registered outside of DNSimple. To do this you must use the `dnssec.rotation_start` and `dnssec.rotation_complete` webhook events. More information is available in [our developer documentation](https://developer.dnsimple.com/v2/webhooks/).
+
+### Automating key rotation with other domain registrars
+
+Our system will deliver a webhook call when DNSSEC rotation is started and completed. You may
```
Comment (itsalyse) [inline content/articles/dnssec.markdown]: >Our system will deliver a webhook call when DNSSEC rotation is started and completed. You may 

You may... ?

Comment (aeden) [reply content/articles/dnssec.markdown]: Extra stuff made it through. I've removed it.

**Comment by aeden** [comment]
@itsalyse please re-review. Thank you.

**Comment by weppos** [comment]
If we can include https://github.com/dnsimple/dnsimple-support/issues/316 and https://github.com/dnsimple/dnsimple-support/issues/195 in this chanceset, it would be great.

**Comment by aeden** [comment]
>  is this supposed to cover #195 ?

No, that is not the intention.

**Comment by aeden** [comment]
> If we can include #316 and #195 in this chanceset, it would be great.

#316 is already present in the article:

dnssec - dnsimple help 2018-12-21 07-05-36

Is there something specific that you want to see done or shall I close #316 ?

I'll add a section to cover #195

**Pull request merged**